src/test/java/io/pravega/connectors/flink/FlinkPravegaTableSinkTest.java

```diff
@@ -98,6 +108,53 @@ public void testBuilder() {
         assertEquals(0, ((FlinkPravegaTableSink.RowBasedRouter) outputFormat.getEventRouter()).getKeyIndex());
     }
 
+    @Test
+    @SuppressWarnings("unchecked")
+    public void testTableSinkDescriptor() {
+        final String cityName = "fruitName";
+        final String total = "count";
+        final String eventTime = "eventTime";
+        final String procTime = "procTime";
+        final String controllerUri = "tcp://localhost:9090";
+        final long delay = 3000L;
+        final String streamName = "test";
+        final String scopeName = "test";
+
+        Stream stream = Stream.of(scopeName, streamName);
+        PravegaConfig pravegaConfig = PravegaConfig.fromDefaults()
+                .withControllerURI(URI.create(controllerUri))
+                .withDefaultScope(scopeName);
+
+        // construct table sink using descriptors and table sink factory
+        Pravega pravega = new Pravega();
+        pravega.tableSinkWriterBuilder()
+                .withRoutingKeyField(cityName)
+                .withWriterMode(PravegaWriterMode.EXACTLY_ONCE)
+                .forStream(stream)
+                .enableMetrics(true)
+                .withPravegaConfig(pravegaConfig);
+
+        final FlinkPravegaTableSourceTest.TestTableDescriptor testDesc = new FlinkPravegaTableSourceTest.TestTableDescriptor(pravega)
+                .withFormat(new Json().failOnMissingField(false) .deriveSchema())
+                .withSchema(
+                        new Schema()
+                                .field(cityName, org.apache.flink.table.api.Types.STRING())
+                                .field(total, org.apache.flink.table.api.Types.DECIMAL())
+                                .field(eventTime, org.apache.flink.table.api.Types.SQL_TIMESTAMP())
+                                .rowtime(new Rowtime()
+                                        .timestampsFromField(eventTime)
+                                        .watermarksFromStrategy(new BoundedOutOfOrderTimestamps(delay))
+                                )
+                                .field(procTime, org.apache.flink.table.api.Types.SQL_TIMESTAMP()).proctime())
```

Comment:
move the closing brace to new line to keep it consistent with other places.

Reply:
fixed